views/uikit_layout.html

```diff
@@ -25,13 +25,24 @@
     <link rel="stylesheet" href="{{=URL('static','uikit-3/css/uikit.min.css')}}" />
     <script src="{{=URL('static','uikit-3/js/uikit.min.js')}}"></script>
     <script src="{{=URL('static','uikit-3/js/uikit-icons.min.js')}}"></script>
+    <script src="{{=URL('static','js/web2py.js')}}"></script>
     <link rel="stylesheet" type="text/css" href="{{=URL('static', 'css/content.css')}}" />
     
+    <!-- include the relevant bits of 'web2py_ajax.html' -->
+    <script type="text/javascript"><!--
+      // These variables are used by the web2py_ajax_init function in web2py_ajax.js (which is loaded below).
+      {{=ASSIGNJS(
+      w2p_ajax_confirm_message = T('Are you sure you want to delete this object?'),
+      w2p_ajax_disable_with_message = T('Working...'),
+      w2p_ajax_date_format = T('%Y-%m-%d'),
+      w2p_ajax_datetime_format = T('%Y-%m-%d %H:%M:%S'),
+      ajax_error_500 = T.M('An error occured, please [[reload %s]] the page') % URL(args=request.args, vars=request.get_vars)
+      )}}
+      //--></script>
     {{
     response.include_meta()
     response.include_files()
     }}
-    <!-- include 'web2py_ajax.html' could be done here, but it performs more than we want so we disect it and keep only the bits we really need -->
```

Comment:
Seems easier to remove calendar.js from ``views/web2py_ajax.html`` rather than replicating it all here. Why it was done like this I've no idea, but it was intentional: https://github.com/OneZoom/OZtree/commit/e4bf7070c449af1ac2f1cf00d037bf4f7d510865

Reply:
I agree. Seems easiest to reinstate with the calendar stuff missing. But I think we probably require the include_meta() and include_files() stuff on all pages, so I'm not sure on the best way to only include web2py.js on *some* of the pages.